.github/workflows/test.yml

```diff
@@ -20,15 +20,5 @@ jobs:
       - uses: actions/checkout@v2
       - name: install
         run: sudo apt-get update && sudo apt-get install aptitude qemu-kvm zstd binutils-dev elfutils libcap-dev libelf-dev libdw-dev python3-docutils
-      - name: Get bpf-next source
-        if: ${{ github.repository != 'kernel-patches/bpf' }}
-        uses: actions/checkout@v2
-        with:
-          repository: kernel-patches/bpf
-          ref: 'bpf-next'
-          path: 'linux'
-      - name: Move linux source in place
-        if: ${{ github.repository != 'kernel-patches/bpf' }}
```

Comment:
we still need this check to make CI runs possible on PRs for kernel-patchs/vmtest. So don't delete it (yet?) until we have a solution for both scenarios. Just invert the condition and hardcode kernel-patches/vmtest repo to keep everything working.

Reply:
Right, I misunderstood your comment on the other issue. I was a bit surprised that we should entirely remove this step :). I'll adjust the PR.